api/HTMLElement.json

```
@@ -2377,25 +2377,45 @@
               "web-features:popover"
             ],
             "support": {
-              "chrome": {
-                "version_added": "133",
-                "partial_implementation": true,
-                "notes": "When using this option, the focus order doesn't change, so the popover does not become the next focus element. See [bug 383343310](https://crbug.com/383343310)."
-              },
+              "chrome": [
+                {
+                  "version_added": "137"
+                },
+                {
+                  "version_added": "133",
+                  "version_removed": "137",
+                  "partial_implementation": true,
+                  "notes": "When using this option, the focus order didn't change, so the popover did not become the next focus element. See [bug 383343310](https://crbug.com/383343310)."
```

Nit: Keep the present tense, even if it applies to older releases.

```suggestion
                  "notes": "When using this option, the focus order doesn't change, so the popover does not become the next focus element. See [bug 383343310](https://crbug.com/383343310)."
```